djconnectwise/models.py

```diff
@@ -2126,6 +2129,20 @@ def __str__(self):
         return self.name
 
 
+class TypeSubTypeItemAssociation(TimeStampedModel):
+    type = models.ForeignKey('Type', blank=True, null=True,
+                             related_name='associations',
+                             on_delete=models.CASCADE)
+    sub_type = models.ForeignKey('SubType', blank=True,
+                                 related_name='associations',
+                                 null=True, on_delete=models.CASCADE)
+    item = models.ForeignKey('Item', blank=True,
+                             related_name='associations',
```

Comment:
Question: Can `related_name`s have the same name?

Reply:
In this case I believe the answer is yes. 

The default `related_name` if I hadn't changed it to `associations` would be `typesubtypeitemassociation_set` and that would have been the same for `type`, `sub_type` and `item`. 

For example:
```
In : st = SubType.objects.first()
In : st
<SubType: Scanner>
In : st.typesubtypeitemassociation_set.all()
<QuerySet []>
In : t.typesubtypeitemassociation_set.all()
<QuerySet []>
In : t
<Type: Break-fix - Professional Services>
In : 
```
My understanding is that the related_name would have to be unique if our model had more than one FK relationship to another model. For example, we would have problems if the model looked like:
```
class TypeSubTypeItemAssociation(TimeStampedModel):
    type_1 = models.ForeignKey('Type'')
    type_2 = models.ForeignKey('Type')
```
type_1 and type_2 would need to have unique related_name's added to differentiate themselves from each other.

Reply:
Thanks for the explanation